### PR TITLE
Kubeapps chart - Lint chart

### PR DESCRIPTION
**Pull request by juan131**
Signed-off-by: juan131 <juan@bitnami.com>

**Description of the change**

- Change chart YAML(s) to make it follow Bitnami standards (and [Helm Best Practices](https://helm.sh/docs/topics/chart_best_practices))
- Some utilities such as **yamllint**, **kubeval** or **helm lint** have been used to ensure there are no syntax errors on the chart's manifests (see https://github.com/bitnami/charts/pull/1452)

### CODE REVIEW

**Comment by juan131** [comment]
> From your description (and looking at some of the changes) this looks like a combination of a mechanical change and some additions.

Yes, it's a combination of "linting changes" and changes to standardise the manifests, so they're consistent with other Bitnami charts.

I can split the diff if you it's very confused for you to review it.

> do you think it would be worth (separately) updating the checks on a PR so that it fails if the yaml/helm lint produce any changes, stopping a PR from being merged?

Yes, I think it can be a great addition. But let's wait until we merge on bitami/charts repo the "final" version of the scripts we want to use to test the charts.

File: chart/kubeapps/values.yaml [hunk 1]
```diff
@@ -83,50 +107,79 @@ frontend:
...
+  ##
   proxypassAccessTokenAsBearer: false
 
-# AppRepository Controller is the controller used to manage the repositories to
-# sync. Set apprepository.initialRepos to configure the initial set of
-# repositories to use when first installing Kubeapps.
+## AppRepository Controller is the controller used to manage the repositories to
+## sync. Set apprepository.initialRepos to configure the initial set of
+## repositories to use when first installing Kubeapps.
+##
```
Comment (andresmgot) [inline chart/kubeapps/values.yaml]: why the empty line in these comments?

File: chart/kubeapps/values.yaml [hunk 2]
```diff
@@ -139,77 +192,144 @@ apprepository:
...
-  # https://github.com/kubeapps/kubeapps/issues/478#issuecomment-422979262
+  #  ca:
+  #  cert:
+  #  key:
+  #  verify: false
+  ## It's possible to modify the default timeout for install/upgrade/rollback/delete apps
+  ## (Default: 300s)
+  # timeout: 300
+  ## Tiller Proxy containers' resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
```
Comment (andresmgot) [inline chart/kubeapps/values.yaml]: this comment is now a bit confusing (because its new place)

File: chart/kubeapps/Chart.yaml [hunk 3]
```diff
@@ -1,21 +1,21 @@
 apiVersion: v1
 name: kubeapps
-version: 3.0.3
+version: 3.0.4
```
Comment (andresmgot) [inline chart/kubeapps/Chart.yaml]: please, revert this change, I plan to release 3.1.0. Bumping the version now will trigger a release but we are fixing a bug that should be added before.

**Comment by absoludity** [comment]
> I can split the diff if you it's very confused for you to review it.

No no :) It's not worth the effort now, I just meant for next time (it's much closer to zero cost at the time). Thanks!

File: chart/kubeapps/values.yaml (hunk 1 above)
Comment (juan131) [reply chart/kubeapps/values.yaml]: It's a standard we're using on our Helm charts for better readability.

File: chart/kubeapps/values.yaml (hunk 2 above)
Comment (juan131) [reply chart/kubeapps/values.yaml]: Do you mean this one?
```yaml
  ## TLS parameters
  tls: {}
  #  ca:
  #  cert:
  #  key:
  #  verify: false
```

Or nesting the comments with the ones for other parameters (e.g. `timeout`, `resources`)?
